### PR TITLE
Initialized React App

### PR DESCRIPTION
**Pull request by rakshit087**
Will clear React Application by next commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
-# Karyakram
-One stop solution for all your meeting needs
+# Getting Started with Create React App
```
Comment (pulkitvyas08) [inline README.md:1]: Remove the default react stuff

File: README.old.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Karyakram
```
Comment (pulkitvyas08) [inline README.old.md:1]: Move this to `README.md`

**Review by pulkitvyas08 — CHANGES_REQUESTED** [review]
Add all the react stuff in a folder called `karyakram-frontend`

**Comment by rakshit087** [comment]
Shall I make a new folder? Bro?

**Comment by pulkitvyas08** [comment]
No, my bad, this is gtg

**Review by pulkitvyas08 — APPROVED** [review]

**Pull request merged**